packages/askui-nodejs/src/lib/download-binaries.ts

```diff
@@ -53,7 +58,7 @@ export function downloadServerBinaries(binaryVersion: string): Promise<void> {
       fs.mkdirSync(binaryFolder, { recursive: true });
     }
 
-    const downloadStream = got.stream(url);
+    const downloadStream = got.stream(url, { ...(proxyAgent ? { agent: proxyAgent } : {}) });
```

Comment:
destructuring seems redundant, what about
```suggestion
    const downloadStream = got.stream(url, proxyAgent ? { agent: proxyAgent } : {});
```